### ISSUE TITLE
FR : list2need : having the ability to set tags that will be attached to all items in the list

### ISSUE THREAD

**Issue by christopheseyler** [useblocks/sphinx-needs#1146]
For example, I need to write down all requirements for a specific component. To easily identify the req associated to the component, I would like to tag them. So, it would be easier to write something like that

```rst
.. list2need::
  :types: req
  :tags: MyComponent
     
   * (REQ-01) Description of REQ1
   * (REQ-02) Description of REQ2
   * (REQ-03) Description of REQ3
```

Where all the 3 requirements would have the MyComponent tag

**Comment by danwos** [comment]
The idea is good.

As a not-so-nice workaround, you could use ``needextend`` to set the tag later.

Untested example:
```rst

My Needs
--------

.. list2need::
  :types: req
  :tags: MyComponent
     
   * (REQ-01) Description of REQ1
   * (REQ-02) Description of REQ2
   * (REQ-03) Description of REQ3

.. needextend:: section_name == "My needs"
   :tags: MyComponent
```

**Comment by christopheseyler** [comment]
Not for now, ... the needextend does not support filter string, only single id (the bug is already reported [#745)](https://github.com/useblocks/sphinx-needs/issues/745)

**Comment by chrisjsewell** [comment]
closed in #296
